src/workerinfo.js

```diff
@@ -2,20 +2,29 @@ const taskcluster = require('taskcluster-client');
 const assert      = require('assert');
 const debug       = require('debug')('workerinfo');
 
+const DAY = 24 * 60 * 60 * 1000;
+
 const ignoreEntityAlreadyExists = (err) => {
   if (!err || err.code !== 'EntityAlreadyExists') {
     throw err;
   }
 };
 
-const expired = entity => new Date(entity.expires) - Date.now() < 24 * 60 * 60 * 1000;
+const expired = entity => new Date(entity.expires) - Date.now() < DAY;
+const shouldUpdateLastDateActive = entity => Date.now() - new Date(entity.lastDateActive) > DAY / 4;
```

Comment:
let's do the subtraction and `<` in the same direction for both :)

Reply:
I think this way is more human readable as it makes more sense. What did you have in mind?

Reply:
I don't mind which way, but my complaint is that one is "somedate - now < something" while the other is "now - somedate > something".  So I had to stare at it a while to figure out that they're both true after somedate + something.  How about

```js
const expired = entity => Date.now() > new Date(entity.expires) + DAY;
const shouldUpdateLastDateActive = entity => Date.now() > new Date(entity.lastDateActive) +  DAY / 4; 
```

Reply:
@djmitche The `expired` suggestion is saying to set `expired` to `true` only when we have gone 1 day past the expiration date. Wouldn't the ideal case be to set `expired` to true if we are within 1 day of the expiration?

Reply:
You're right.  I flipped the sign on the inequality correctly.  See, it is confusing!

```js
const expired = entity => Date.now() > new Date(entity.expires) - DAY;
```